graph.go

```diff
@@ -195,16 +195,21 @@ func (c *GraphCmd) Execute(args []string) error {
 }
 
 func relPath(base, path string) string {
-	rp, err := filepath.Rel(base, path)
+	pathWithoutSymlinks, err := filepath.EvalSymlinks(path)
```

Comment:
If you don't do this, then you'll have `.srclib-gopath/src/github.com/gorilla/mux/mux.go` instead of `mux.go` in the filepaths. This is not about the repo being symlinked, but about the symlink inside of `.srclib-gopath`.